Modules/Microsoft365DSC/Examples/Resources/AADHomeRealmDiscoveryPolicy/1-Create.ps1

```diff
@@ -0,0 +1,43 @@
+<#
+This example is used to test new resources and showcase the usage of new resources being worked on.
+It is not meant to use as a production baseline.
+#>
+
+Configuration Example
+{
+    param(
+        [Parameter()]
+        [System.String]
+        $ApplicationId,
+
+        [Parameter()]
+        [System.String]
+        $TenantId,
+
+        [Parameter()]
+        [System.String]
+        $CertificateThumbprint
+    )
+    Import-DscResource -ModuleName Microsoft365DSC
+    node localhost
+    {
+        AADHomeRealmDiscoveryPolicy "AADHomeRealmDiscoveryPolicy-displayName-value"
+        {
+            ApplicationId         = $ConfigurationData.NonNodeData.ApplicationId;
```

Comment:
This should only be $ApplicationId and $CertificateThumbprint

Reply:
updating this and all below